### DISCUSSION TITLE
Installation failed

### DISCUSSION THREAD

**Discussion by phengchen** [WASasquatch/was-node-suite-comfyui#360] [General]
ModuleNotFoundError: No module named 'distutils'

**Comment by VLevithan** [comment]
There's a solution to the problem here: https://stackoverflow.com/questions/77233855/why-did-i-got-an-error-modulenotfounderror-no-module-named-distutils
"The problem is that Python 3.12 does not come with a stdlib distutils module (changelog), because distutils was deprecated in 3.10 and removed in 3.12. See PEP 632 – Deprecate distutils module.

You can still use distutils on Python 3.12+ by installing setuptools.

When that doesn't work, you may need stay on Python < 3.12 until the 3rd-party package (skfuzzy in this case) publishes an updated release for Python 3.12 support"
